lib/src/facts/linux/os_linux.cc

```diff
@@ -84,7 +84,8 @@ namespace facter { namespace facts { namespace linux {
                 make_tuple(boost::regex("(?i)scientific linux CERN"),         string(os::scientific_cern)),
                 make_tuple(boost::regex("(?i)scientific linux release"),      string(os::scientific)),
                 make_tuple(boost::regex("(?im)^cloudlinux"),                  string(os::cloud_linux)),
-                make_tuple(boost::regex("(?im)^virtuozzo linux"),             string(os::virtuozzo_linux)),
+                make_tuple(boost::regex("(?im)^virtuozzo linux"),             string(os::virtuozzo_linux6)),
+                make_tuple(boost::regex("(?im)^virtuozzo linux"),             string(os::virtuozzo_linux7)),
```

Comment:
I need to re-do this PR, because I got the way `facter` collects this fact wrong. I thought it's matching `VirtuozzoLinux`. So, actually it's correctly reporting `"operatingsystem": "VirtuozzoLinux"` on VZ7.
```
# facter os
{
  architecture => "x86_64",
  family => "Linux",
  hardware => "x86_64",
  name => "VirtuozzoLinux",
  release => {
    full => "3.10.0-862.9.1.vz7.63.3",
    major => "3",
    minor => "10"
  },
  selinux => {
    enabled => false
  }
}
```
When the changes branan requested are implemented, this PR will introduce support for *PCS* and not *VZ7*.

(Another issue is `facter` falling back to parsing the kernel version for VZ7.)